### PR TITLE
Fixed Close-SmbOpenFile examples

### PR DESCRIPTION
**Pull request by dariomws**
Related to https://github.com/MicrosoftDocs/windows-powershell-docs/pull/2360.

### CODE REVIEW

File: docset/winserver2019-ps/smbshare/Close-SmbOpenFile.md [hunk 1]
```diff
@@ -88,8 +92,8 @@ Performing operation 'Close-File' on Target '4415226383569'.
 This command closes one or more files that are open by one of the client identified with the session ID 4415226380393 of the SMB server.
```
Comment (JohanFreelancer9) [inline docset/winserver2019-ps/smbshare/Close-SmbOpenFile.md:92]:
```suggestion
This command closes one or more files that are open by one of the clients identified with the session ID 4415226380393 of the SMB server.
```

Comment (JohanFreelancer9) [reply docset/winserver2019-ps/smbshare/Close-SmbOpenFile.md:92]: Not related to this PR, but please also commit this minor typo.

File: docset/winserver2022-ps/smbshare/Close-SmbOpenFile.md [hunk 2]
```diff
@@ -88,8 +92,8 @@ Performing operation 'Close-File' on Target '4415226383569'.
 This command closes one or more files that are open by one of the client identified with the session ID 4415226380393 of the SMB server.
```
Comment (JohanFreelancer9) [inline docset/winserver2022-ps/smbshare/Close-SmbOpenFile.md:92]:
```suggestion
This command closes one or more files that are open by one of the clients identified with the session ID 4415226380393 of the SMB server.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Review by JohanFreelancer9 — APPROVED** [review]

**Review by sdwheeler — APPROVED** [review]

**Pull request merged**